checkita-core/src/main/resources/db/specific/h2/V1.1__create_job_state.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "${defaultSchema}"."results_job_config"
```

Comment:
Lets name this table as `job_state` (here and in files for other DBs) as it is not kind of result. I suppose that would be more appropriate as this table  will store job configuration states for which the results were computed.